.github/workflows/test_rust.yml

```diff
@@ -94,3 +91,23 @@ jobs:
         with:
           name: swf_images
           path: tests/**/*.png.updated
+
+  check-required:
+    needs: changes
+    if: needs.changes.outputs.src == 'false'
+    name: Test Rust ${{ matrix.rust_version }} / ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.rust_version == 'nightly' || matrix.rust_version == 'beta' }}
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
It might be a little silly, but looks like YAML supports "[anchors](https://yaml.org/spec/1.2.1/#id2785586)", which allow reuse of previous nodes.

Paste the following in [YAML playground](https://eemeli.org/yaml-playground/):
```yml
first:
  matrix: &matrix
    rust_version: [stable]
    os: [ubuntu-latest, windows-latest, macOS-latest]
    include:
      - rust_version: nightly
        os: ubuntu-latest
      - rust_version: beta
        os: ubuntu-latest

second:
  matrix: *matrix
```

Reply:
Would be extremely neat if this works! I'll check after work.

Reply:
Unfortunately not: https://github.com/midgleyc/ruffle/actions/runs/1254280793

> The workflow is not valid. .github/workflows/test_rust.yml: Anchors are not currently supported. Remove the anchor 'strategy'